### PR TITLE
Fix osu! brand naming

### PR DESCRIPTION
**Pull request by Joehuu**

### CODE REVIEW

File: resources/lang/en/community.php [hunk 1]
```diff
@@ -23,13 +23,13 @@
         'header' => [
             // size in font-size
             'big_description' => 'Love osu!?<br/>
-                                Support the osu! dev :D',
+                                Support the osu!dev :D',
```
Comment (peppy) [inline resources/lang/en/community.php]: should probably just be "Support the development of osu! :D" or something similar.

File: resources/lang/en/layout.php [hunk 2]
```diff
@@ -180,7 +180,7 @@
         ],
         'support' => [
```
Comment (peppy) [inline resources/lang/en/layout.php]: is this footer only shown when logged out? if so we can probably just remove this section.

**Comment by Joehuu** [comment]
I have a question. Shouldn't the title of the forums be `osu! forums` not `osu! community`.

**Comment by peppy** [comment]
Yeah, it may be best to rename it now that "community" is a top level header.

**Comment by Joehuu** [comment]
The three other pages under community are not consistent in terms of wording. I think all of them should have `osu!` in front of them, and remove `community` altogether.

I'm also confused about the capitalisation after osu!. Should it be `osu! forums` or `osu! Forums`? Check the others listed.



`osu! Community Contests` -> `osu! Contests`?



`Community Tournaments` -> `osu! Tournaments`?



`Live Streams` -> `osu! Live Streams`?

**Comment by peppy** [comment]
I don't think we need the 'osu!' prefix before "Community Contests"

**Pull request merged**